spec/integration/jara_spec.rb

```diff
@@ -129,7 +136,13 @@ def run_package(project_dir, environment='production')
         end
 
         before :all do
-          isolated_run(@test_project_dir, 'git checkout -b staging', 'echo "puts \"Hello staging\"" > bin/check', 'git add .', 'git commit -m "Change the check message"', 'git push -u origin staging')
+          isolated_run(@test_project_dir,
+            'git checkout -b staging',
+            'echo "puts \"Hello staging\"" > bin/check',
+            'git add .',
+            'git commit -m "Change the check message"',
+            'git push -u origin staging'
+          )
```

Comment:
This indentation looks a bit weird to me; I'd align the second argument with the first.